sopel/cli/utils.py

```diff
@@ -222,7 +222,11 @@ def add_common_arguments(parser):
         default=None,
         metavar='filename',
         dest='config',
-        help='Use a specific configuration file')
+        help='Use a specific configuration file. '
+             'A config name can be given and the configuration file will be '
+             'find in Sopel\'s homedir (default to ``~/.sopel/default.cfg``). '
```

Comment:
```suggestion
             'found in Sopel\'s homedir (defaults to ``~/.sopel/default.cfg``). '
```

Also, doesn't the option actually default to `default`, rather than that full path?

Reply:
The option itself defaults to `None` indeed, but its usage is to default to "the default" config path.